dodo.py

```diff
@@ -268,18 +275,181 @@ def client(db_conn_string, command, args):
     }
 
 
+def task_behavior():
+    """
+    Behavior modeling: datagen, plan data differencing, and OU model training.
+    """
+
+    def run_behavior(all, datagen, diff, train, build_pg, build_benchbase):
```

Comment:
I had them separate originally but rolled them into the behavior task so it didn't mislead anyone into assuming that it would provide Postgres/BenchBase ready to use for non-behavior system components.  How would you like me to factor the tasks for building postgres and benchbase in light of the fact that action recommendation, workload forecasting, and behavior modeling each interact with Postgres in their own way?